### PR TITLE
Window params

### PR DESCRIPTION
**Pull request by mamarcus64**
## Description
Convert some window params to the new ros2 standards.

## Associated Issue
Issue #1601

### CODE REVIEW

**Review by HussainGynai — APPROVED** [review]
LGTM

File: soccer/src/soccer/window_evaluator.hpp [hunk 1]
```diff
@@ -4,9 +4,15 @@
 
 #include <rj_geometry/segment.hpp>
 #include <rj_geometry/point.hpp>
+#include <rj_param_utils/param.hpp>
 #include "robot.hpp"
 #include "system_state.hpp"
 
+constexpr auto windowConfigParamModule = "optimization";
```
Comment (kylestach) [inline soccer/src/soccer/window_evaluator.hpp]: Nit: style guide says `kWindowConfigParamModule`

File: soccer/src/soccer/window_evaluator.hpp [hunk 2]
```diff
@@ -4,9 +4,15 @@
...
+
+    DECLARE_NS_FLOAT64(windowConfigParamModule, optimization, angle_score_coefficient)
+    DECLARE_NS_FLOAT64(windowConfigParamModule, optimization, distance_score_coefficient)
```
Comment (kylestach) [inline soccer/src/soccer/window_evaluator.hpp]: Nit: indentation (run `make pretty-lines` to fix all formatting issues)

File: soccer/src/soccer/window_evaluator.cpp [hunk 3]
```diff
@@ -13,16 +13,17 @@ REGISTER_CONFIGURABLE(WindowEvaluator)
 
 using namespace rj_geometry;
 
-ConfigDouble* WindowEvaluator::angle_score_coefficient;
-ConfigDouble* WindowEvaluator::distance_score_coefficient;
+DEFINE_NS_FLOAT64(windowConfigParamModule, optimization, angle_score_coefficient, 0.7,
+	"Angle score coefficient.");
```
Comment (kylestach) [inline soccer/src/soccer/window_evaluator.cpp]: Do we not actually use this variable anywhere?

**Review by kylestach — CHANGES_REQUESTED** [review]
Looks good, some minor naming nits.

File: soccer/src/soccer/window_evaluator.cpp (hunk 3 above)
Comment (mamarcus64) [reply soccer/src/soccer/window_evaluator.cpp]: We don't. Should I leave it in, since there's a value associated with it?

**Review by kylestach — APPROVED** [review]

**Pull request merged**